source/Geometry/src/Layer.cc

```diff
@@ -24,12 +26,12 @@ Layer::Layer(int id, int barrelNumLayers, bool sameRods, bool barrelMinRFixed, b
  sameParityRods (       "sameParityRods"     , parsedAndChecked(), true),
  layerRotation  (       "layerRotation"      , parsedOnly()      , 0.),
  tiltedLayerSpecFile(   "tiltedLayerSpecFile", parsedOnly()),
- m_smallDelta   (       "smallDelta"         , parsedAndChecked()),
+ smallDelta     (       "smallDelta"         , parsedAndChecked()),
  m_smallParity  (       "smallParity"        , parsedAndChecked(),-1),
```

Comment:
The same comment as for the m_bigDelta() applies here, we simply need to agree on the "coding" conventions for this special variables, which also work as functions ...